perftests/network-ssl/test_network_ssl.sh

```diff
@@ -4,7 +4,7 @@ failed=0
 output=test_network_ssl_stderr.log
 
 # Check for online access: can we access badssl.com?
-if ! echo -e "GET / HTTP/1.0\n\n" | nc badssl.com 80 -w 10 > /dev/null 2>&1
+if ! printf "GET / HTTP/1.0\n\n\n" | nc badssl.com 80 -w 10 > /dev/null 2>&1
```

Comment:
You need two EOLs at the end of an HTTP request, not three.  Also, you should have "\r\n\r\n" since the EOL in HTTP is CRLF.

Not that it really matters if we're just checking that we can connect to the host, but still...